### PR TITLE
Minorfix

### PR DESCRIPTION
**Pull request by xiaoxiang781216**
Found when porting to NuttX

### CODE REVIEW

File: lib/asyn-thread.c [hunk 1]
```diff
@@ -667,7 +667,7 @@ int Curl_resolver_getsock(struct connectdata *conn,
 #endif
     ms = Curl_timediff(Curl_now(), reslv->start);
     if(ms < 3)
-      milli = 0;
+      milli = ms;
```
Comment (jay) [inline lib/asyn-thread.c]: why this?

File: lib/curl_setup_once.h [hunk 2]
```diff
@@ -374,6 +374,7 @@ typedef int sig_atomic_t;
  * Macro used to include assertion code only in debug builds.
  */
 
+#undef DEBUGASSERT
 #if defined(DEBUGBUILD) && defined(HAVE_ASSERT_H)
 #define DEBUGASSERT(x) assert(x)
```
Comment (jay) [inline lib/curl_setup_once.h]: Where is DEBUGASSERT already defined?

File: lib/asyn-thread.c (hunk 1 above)
Comment (xiaoxiang781216) [reply lib/asyn-thread.c]: Sorry, we have a better fix. I remove this patch from pull request, pleaase ignore it.

File: lib/curl_setup_once.h (hunk 2 above)
Comment (xiaoxiang781216) [reply lib/curl_setup_once.h]: Many OS or library define DEBUGASSERT, you can see here:
https://github.com/search?l=C&p=1&q=DEBUGASSERT&type=Code
One I really hit is:
https://github.com/apache/incubator-nuttx/blob/master/include/assert.h
Anyway, it isn't harmful to undefine it before define.

**Comment by jay** [comment]
Thanks. You may want to investigate why CURL_SIZEOF_CURL_OFF_T is >= 4. In most cases CURL_SIZEOF_CURL_OFF_T should be 8 because most compilers can support 64-bit curl_off_t even if the platform is 32-bit.

**Pull request closed without merging**